<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1452</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>1271</width>
      <height>791</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QPushButton" name="PRINT">
       <property name="text">
        <string>PRINT</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QTableWidget" name="result">
       <property name="minimumSize">
        <size>
         <width>1129</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>10</number>
       </property>
       <property name="midLineWidth">
        <number>10</number>
       </property>
       <property name="rowCount">
        <number>11</number>
       </property>
       <property name="columnCount">
        <number>6</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>29</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>Назва автомобільної дороги</string>
        </property>
        <property name="foreground">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>Тип</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>Протяжність</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>Кількість смуг</string>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string> Наявність пішохідної доріжки </string>
        </property>
       </item>
       <item row="0" column="5">
        <property name="text">
         <string>Наявність розділювача посередині дороги</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="8" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="8" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="8" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="8" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="8" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QPushButton" name="Task5">
       <property name="text">
        <string> 5Визначити автомобільні дороги з найбільшою кількістю смуг та наявними пішохідними доріжками які належать до регіональних.</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="scan">
       <property name="text">
        <string>SCANFROMFILE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTextEdit" name="textEdit">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;blablablaAKuix-Ode,a,9,6,+,+&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;BKuix-Ode,a,8,3,-,-&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;CKuix-Ode,b,7,2,+,+&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;DKuix-Ode,c,6,6,+,+&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;EKuix-Ode,b,5,6,+,+&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;FKuix-Ode,c,4,6,+,+&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;GKuix-Ode,a,3,6,+,+&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;EKuix-Ode,b,2,6,+,-&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Huix-Ode,a,1,6,-,+&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="Task3">
       <property name="text">
        <string>3 Знайти всі дороги, в яких наявні розділювачі посередині, кількість смуг &gt;2 та згрупувати за типом.</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QTextEdit" name="max">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>1Впорядкувати дороги за протяжністю</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="MAX">
       <property name="text">
        <string>2 Знайти найкоротшу дорогу, де найбільша кількість смуг.</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QPushButton" name="TAsk4">
       <property name="text">
        <string>4 Визначити типи автомобільних доріг, з найбільшою протяжністю та наявністю пішохідних доріжок.</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="tmmpp">
       <property name="text">
        <string>CHECK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="tmp"/>
     </item>
     <item row="9" column="0">
      <widget class="QTableWidget" name="information">
       <property name="minimumSize">
        <size>
         <width>1129</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>10</number>
       </property>
       <property name="midLineWidth">
        <number>10</number>
       </property>
       <property name="rowCount">
        <number>11</number>
       </property>
       <property name="columnCount">
        <number>6</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>29</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <column/>
       <item row="0" column="0">
        <property name="text">
         <string>Назва автомобільної дороги</string>
        </property>
        <property name="foreground">
         <brush brushstyle="NoBrush">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>Тип</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>Протяжність</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>Кількість смуг</string>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string> Наявність пішохідної доріжки </string>
        </property>
       </item>
       <item row="0" column="5">
        <property name="text">
         <string>Наявність розділювача посередині дороги</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="1" column="4">
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item row="1" column="5">
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="4">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="2" column="5">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="4">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="3" column="5">
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string>123</string>
        </property>
       </item>
       <item row="4" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="4">
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item row="4" column="5">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="1">
        <property name="text">
         <string>b</string>
        </property>
       </item>
       <item row="5" column="2">
        <property name="text">
         <string>237</string>
        </property>
       </item>
       <item row="5" column="3">
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item row="5" column="4">
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item row="5" column="5">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>Kiuy-Koval</string>
        </property>
       </item>
       <item row="6" column="1">
        <property name="text">
         <string>c</string>
        </property>
       </item>
       <item row="6" column="2">
        <property name="text">
         <string>43</string>
        </property>
       </item>
       <item row="6" column="3">
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item row="6" column="4">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="6" column="5">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string>Lviiv-Dnipro</string>
        </property>
       </item>
       <item row="7" column="1">
        <property name="text">
         <string>a</string>
        </property>
       </item>
       <item row="7" column="2">
        <property name="text">
         <string>280</string>
        </property>
       </item>
       <item row="7" column="3">
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item row="7" column="4">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="7" column="5">
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>Odessa-Oleh</string>
        </property>
       </item>
       <item row="8" column="1">
        <property name="text">
         <string>b</string>
        </property>
       </item>
       <item row="8" column="2">
        <property name="text">
         <string>250</string>
        </property>
       </item>
       <item row="8" column="3">
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item row="8" column="4">
        <property name="text">
         <string>+</string>
        </property>
       </item>
       <item row="8" column="5">
        <property name="text">
         <string>-</string>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="2">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="3">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="4">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="10" column="5">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="rbtable">
         <property name="text">
          <string>Дані з таблиці</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbfile">
         <property name="text">
          <string>дані з файлу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbwritten">
         <property name="text">
          <string>дані введені</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1452</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
